test/functional/kernel/hyperplane/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+###############################################################################
+# Copyright (c) 2016-23, Lawrence Livermore National Security, LLC
+# and RAJA project contributors. See the RAJA/LICENSE file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+###############################################################################
+
+#
+# Generate kernel hyperplane tests for each enabled RAJA back-end.
+#
+set(TESTTYPES 2D 3D)
+
+foreach( BACKEND ${KERNEL_BACKENDS} )
+  foreach( TEST_TYPE ${TESTTYPES} )
+    # Removing Sycl backend, implementation of Hyperplane does not exist
+    if( NOT ((BACKEND STREQUAL "Sycl")) )
```

Comment:
Do we need to setup an issue for this?

Reply:
We could, but I don't think it's necessary at the moment. There aren't any use cases for this in our applications, and I don't think there are any long term plans for running on this backend. We can create the issue, and work on it as demand arises?

Reply:
Brian H. didn't address it (yet) in his SYCL back-end work because we didn't have test for it in our current test organization. I know Verinder wants to run Ardra on Aurora EA system if he can. We probably should add an issue to support it with SYCL so we don't forget about it.